### PR TITLE
add storageproviderids to config

### PR DESCRIPTION
**Pull request by butonic**
- [ ] update reva to https://github.com/cs3org/reva/pull/2790 or https://github.com/cs3org/reva/pull/2792
cc @kobergj

### CODE REVIEW

File: extensions/gateway/pkg/command/command.go [hunk 1]
```diff
@@ -223,6 +224,7 @@ func spacesProviders(cfg *config.Config, logger log.Logger) map[string]map[strin
 	// generate rules based on default config
 	return map[string]map[string]interface{}{
 		cfg.StorageUsersEndpoint: {
+			"providerid": "1284d238-aa92-42ce-bdc4-0b0000009157",
```
Comment (butonic) [inline extensions/gateway/pkg/command/command.go:227]: for now this is hardcoded ... we should generate the storageproviderid for the userstorageprovider on first start... furthermore, the userstorageprovider currently also stores project spaces ... so maybe just call it `storageprovider`?

Comment (butonic) [reply extensions/gateway/pkg/command/command.go:227]: maybe with `ocis init` from https://github.com/owncloud/ocis/pull/3551

Comment (kobergj) [reply extensions/gateway/pkg/command/command.go:227]: Should we move it to `utils` package meanwhile? Then it is aligned with the `StorageProviderID` and `PublicStorageProviderID`?

Comment (butonic) [reply extensions/gateway/pkg/command/command.go:227]: The userstorageproviderid is not used in the reva codebase, only in config files, so there is no point in adding a variable to revas `pkg/utils` or `resourceid` package that is unused.

**Pull request merged**